doc/content/source/postprocessor/AveragePercentExtension.md

```diff
@@ -0,0 +1,21 @@
+# AveragePercentExtension
+
+!syntax description /Postprocessors/AveragePercentExtension
+
+## Description
+
+This `PostProcessor` computes the average percentage extension between two sets of points specified in two arrays defined with the `first_point` and `last_point` parameters. The point positions account for the current displacements, so this distance will evolve over time for a deforming body. If the separation between two specific points is desired, single points rather than a arrays of points can be specified.
+
+This `PostProcessor` computes average percentage extension between a set of points specified in two array `first_point` and `last_point`.
```

Comment:
between two sets of points specified in two arrays:

Also, it's probably also worth stating that: The extensions between the points with matching positions in these arrays are computed, and then their average is computed. The lengths of these sets of points must be the same.